### PR TITLE
Add key retry

### PR DESCRIPTION
**Pull request by jprzychodzen**

### CODE REVIEW

File: templates/Dockerfile.template [hunk 1]
```diff
@@ -12,7 +12,20 @@ RUN set -x \
 	&& wget -q -O /usr/local/src/gosu.tar.gz "https://github.com/tianon/gosu/archive/$GOSU_VERSION.tar.gz" \
 # extract gosu binary and check signature
 	&& export GNUPGHOME="$(mktemp -d)" \
-	&& gpg --keyserver pgp.mit.edu --recv-keys {{ $gosu.Gpg }} \
+	&& for server in \
```
Comment (wgrzelak) [inline templates/Dockerfile.template]: Add `&& found='' \` before this line.

File: templates/Dockerfile.template [hunk 2]
```diff
@@ -12,7 +12,20 @@ RUN set -x \
...
+		ha.pool.sks-keyservers.net \
+		pgp.mit.edu \
+		na.pool.sks-keyservers.net \
+		eu.pool.sks-keyservers.net \
+		oc.pool.sks-keyservers.net \
+		ha.pool.sks-keyservers.net \
+		hkp://p80.pool.sks-keyservers.net:80 \
+		hkp://keyserver.ubuntu.com:80 \
+	; do \
+		gpg --keyserver pgp.mit.edu --recv-keys {{ $gosu.Gpg }} \
```
Comment (wgrzelak) [inline templates/Dockerfile.template]: s/pgp.mit.edu/$server/

File: templates/Dockerfile.template [hunk 3]
```diff
@@ -23,7 +36,20 @@ RUN set -ex; \
 # https://artifacts.elastic.co/GPG-KEY-elasticsearch
 	key='{{ $elasticsearch.Gpg }}'; \
 	export GNUPGHOME="$(mktemp -d)"; \
-	gpg --keyserver pgp.mit.edu --recv-keys "$key"; \
+	for server in \
```
Comment (wgrzelak) [inline templates/Dockerfile.template]: Add `&& found='' \` before this line.

Comment (jprzychodzen) [reply templates/Dockerfile.template]: Good catch!

**Pull request merged**